src/acl/Ip.cc

```diff
@@ -13,6 +13,7 @@
 #include "acl/Ip.h"
 #include "cache_cf.h"
 #include "ConfigParser.h"
+#include "debug/Messages.h"
 #include "debug/Stream.h"
```

Comment:
`debug/Message.h` imports `debug/Stream.h` no need to have both.
```suggestion
#include "debug/Messages.h"
```

Reply:
Why do you insist on these changes? I do not think they are a good idea: The users of header A are not supposed to know what other headers A includes -- the `#include` list of header A is an internal business of header A (despite the fact that C++ cannot make that information private or hidden from A users). That include list may change at any time, depending on internal A needs, and those changes should not affect any users (that use unaffected header A API). 

The minute time savings during compilation are not worth the inconveniences this "know what the included headers include and optimize to include once" approach causes IMO.

Please reconsider.

Reply:
Beyond the precompiler work, it simplifies the dependency graphs in code documentation. Those graphs can be complicated enough without unnecessary edges causing the graph generator to shuffle object representations around to fit them in.
This is a weak insistence - I do not expect people to remember or even check every include list, but please do when it is known (like now). Eventually we should get this check automated and not have to care.

Reply:
Thank you for the change despite your disagreement.

Reply:
I had no choice in the matter.